docs/api/session-management.asciidoc

```diff
@@ -0,0 +1,11 @@
+[role="xpack"]
+[[session-management-api]]
```

Comment:
> Will you be adding more APIs under the "Kibana user session management" section?

We don't have capacity yet, but I can see that we'll want to add user session enumeration APIs that will support session management UI and automation workflows for the admins in the future.

> Is Kibana needed? Or, can the title simply be "User session management APIs"?

I'm not sure to be honest. User session is a Kibana-only thing and I basically used the same convention we used for Spaces that is also a Kibana-only thing (`Kibana Spaces APIs`). I think `Kibana` is redundant here since this is a section of the Kibana docs after all, but what do you think?

Reply:
I think Kibana is not needed.